ui/app/components/not-found.js

```diff
@@ -4,14 +4,12 @@
  */
 
 import { inject as service } from '@ember/service';
-import { alias } from '@ember/object/computed';
-import Component from '@ember/component';
+import Component from '@glimmer/component';
```

Comment:
Could we add some jsdocs here so we know what args and what the component does? With Claire's docify work this would be handy!